ingestors/kafka/src/main/scala/hydra/kafka/algebras/KafkaClientAlgebra.scala

```diff
@@ -89,129 +95,183 @@ object KafkaClientAlgebra {
 
   object PublishError {
 
-    final case object Timeout
+    case object Timeout
```

Comment:
Would it be worth adding the topic name with the error message. I agree there is no need for a stack trace.